src/cma.ts

```diff
@@ -225,26 +156,45 @@ async function getCmaOutput(readLine, errorObj) {
   });
 }
 
+/**
+ * Invoke the task function and wrap the result as a Promise
+ *
+ * @param {Function} TaskFunction - the task function to be invoked
+ * @param {Object} cumulusMessage - a full Cumulus message
+ * @param {Object} context - the Lambda context
+ * @returns {Promise} - the result of invoking the task function
+ */
+function invokePromisedTaskFunction(
```

Comment:
Because we are using `await invokePromisedTaskFunction(...)` when this is actually used, this function is not needed. `await` will work properly if it is given a Promise _or_ a non-Promise.

Reply:
Right you are - this was left in when we made everything async.    Reading comprehension fail last evening. 

4b8b049